teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1923 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #71.            except ValueError:
Line #72.                # Performance can only be plotted for the left out target in LOTO split
Line #73.                pass
</pre><p>Would it make sense to you if we added the comment here as print statement (informing the user why there is no output in some cases)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
<p>Definitely</p>